### ISSUE TITLE
Attempt to use nvQuickSite on Server 2019 / SQL 14+

### ISSUE THREAD

**Issue by alan2smith** [nvisionative/nvQuickSite#234]
I have made several attempts to install nvQuickSite on Windows Server 2019 / SQL 14. I have tried target directories in c:\inetpub\wwwroot and at c:\DNN with different host names. Error is always the same 'IIS Web Core' , This configuration section cannot be use as a path... 
SQL Databases' appear to have been created but I get the error when I try to load initial page from browser.

Any Ideas? Thank you

Alan Smith

**Comment by alan2smith** [comment]
I resolved the issue partially. 
The server needed features added to "Web Server" for Application Development. I selected: .Net Ext. 3.5, .Net Ext. 4.7, ASP.NET 3.5, ASP.NET 4.7, ISAPI Ext., ISAPI Filters and WebSocket Protocol.

I now get the installation page but it says: "Database Connection Error" / "No valid database connection detected. Standard Database setup option is unavailable. Changing from "Default" to "Custom" does nothing.  (Tried DNN v9.3.2 and v9.2.2 same error)

**Does SQL 14 need a different connection string ?**
<add name="SiteSqlServer" connectionString="Server=(local);Database=DNNTest;Integrated Security=True;" providerName="System.Data.SqlClient" />

**Comment by nvisionative** [comment]
@alan2smith you may want to take a look at an article by @CBPSC on IIS configuration.  While this is based on an older version of IIS, it is essentially the same setup.

Regarding support for Windows Server 2019, officially we have not tested it on there.  Here are the [official minimum requirements](https://github.com/nvisionative/nvQuickSite/wiki/Minimum-System-Requirements) but we would be very interested in your results.  😄 👍

**Comment by alan2smith** [comment]
Update: The reason I was getting the SQL connection error was IE 11. Microsoft has only one supported browser in Windows Server 2019 (this may be the case for previous 'Server' versions). I installed Firefox and the installation was able to complete as expected.
I had already removed and re-installed SQL express 2017 (local admin vs domain admin) finally replacing it with SQL Express 2016 but the error appears to be unrelated to the SQL version and was the browser.
I am not going to re-install SQL express 2017 now that I have the server running.
Thank you
